models/CoronSegmentator/scripts/usd_design/usd_mesh_creator.py

```
@@ -0,0 +1,179 @@
+                UsdGeom.XformOp.TypeRotateX,
+                UsdGeom.XformOp.PrecisionFloat,
+                "rotate_root",
+            ).Set(-90)
+
+            # Load meshes in parallel
+            with concurrent.futures.ThreadPoolExecutor() as executor:
+                meshes = list(executor.map(self.load_mesh, self.input_files))
+            # meshes = [self.load_mesh(file) for file in self.input_files]
+
+            mesh_paths = []
+            for input_file, mesh in zip(self.input_files, meshes):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)